test/claimsTests/createIdentityTests.ts

```diff
@@ -0,0 +1,27 @@
+import { IAM } from "../../src/iam";
+import { createIam, rootOwner } from "../iam.test";
+import { rpcUrl } from "../setup_contracts";
+
+export const createIdentityTests = () => {
+    let rootOwnerIam: IAM;
+
+    beforeAll(async () => {
+        rootOwnerIam = await createIam(rootOwner.privateKey, { createDocument: true });
+    });
+
+    test("Identity token should be defined", async () => {
+        const identityToken = await rootOwnerIam.createIdentityProof();
+        expect(identityToken).toBeDefined;
+    });
+
+    test("Identity token should be defined for delegate", async () => {
+        const ownerAddress = await rootOwner.getAddress();
+        const identityProofDid = `did:ethr:${ownerAddress}`;
+        const delegateIdentityToken = await rootOwnerIam.createDelegateProof(
+            rootOwner.privateKey,
+            rpcUrl,
+            identityProofDid,
+        );
+        expect(delegateIdentityToken).toBeDefined;
+    });
```

Comment:
I think best case scenario we could also test that these tokens verify, But I think that might be tough without further mocking IPFS. @knzeng-e Before merging, could you `npm link` this with passport-did-auth to check that verification works if you use these methods? Does that make sense?

Reply:
if I get you correctly @jrhender, you suggest that I link passport-did-auth and write some tests using those two methods in passport to ensure the verification process ?